test/local/metrics/context.js

```diff
@@ -943,14 +935,13 @@ describe('metricsContext', () => {
       assert.ok(mockLog.warn.calledWithExactly({
         op: 'metrics.context.validate',
         valid: false,
-        reason: 'invalid signature',
-        agent: 'Firefox'
+        reason: 'invalid signature'
       }), 'log.warn was called with the expected log data')
     }
   )
 
   it(
-    'metricsContext.validate with flow signature from different user agent',
+    'metricsContext.validate with flow signature including user agent',
```

Comment:
Tweaked this test to be more useful, so the user agent string now matches the generated signature but we assert that fails because the fallback has been taken away.

Reply:
I'm going to try to rephrase because I'm trying to understand. This test previously had an expected failure because the user-agent changed from that used to create the HMAC. Now, the teat fails even though the user-agent is the same because the user-agent is no longer used to create the HMAC?

The reason seems like it'd be a useful comment as to why the test expects a failure?